ci-operator/config/openshift/special-resource-operator/openshift-special-resource-operator-master.yaml

```diff
@@ -87,6 +86,54 @@ tests:
   commands: make verify
   container:
     from: src
+- as: disconnected-build
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.10"
+  run_if_changed: ^Dockerfile$
+  steps:
+    test:
+    - as: create-podman-container
+      cli: latest
+      commands: |
+        oc -n openshift-config  get secret/pull-secret -o yaml | sed 's/openshift-config/default/' | sed 's/\.dockerconfigjson/auth.json/' | grep -v type: | oc apply -f -
+        oc -n default apply -f ci/podman.yaml
+        oc -n default wait pod/podman --for=condition=Ready
```

Comment:
```suggestion
        oc apply -f ci/podman.yaml
        oc wait pod/podman --for=condition=Ready
```

Reply:
The `oc` client is set to the namespaces assigned for the job which doesn't exist in ephemeral cluster.
Removing the `-n default` will result in an error from the `oc` client that can't find the requested namespace.